src/main/IO/Process/Services/Avatars/AvatarSettingsRepository.cs

```diff
@@ -0,0 +1,300 @@
+﻿using ei8.Avatar.Installer.Domain.Model.Avatars.Settings;
+using ei8.Avatar.Installer.Domain.Model;
+using neurUL.Common.Domain.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ei8.Avatar.Installer.Domain.Model.Avatars;
+using ei8.Avatar.Installer.Common;
+
+namespace ei8.Avatar.Installer.IO.Process.Services.Avatars;
+
+public class AvatarSettingsRepository : IAvatarSettingsRepository
+{
+    private readonly IAvatarContextService avatarContextService;
+
+    public AvatarSettingsRepository(IAvatarContextService avatarContextService)
+    {
+        AssertionConcern.AssertArgumentNotNull(avatarContextService, nameof(avatarContextService));
+
+        this.avatarContextService = avatarContextService;
+    }
+
+    public async Task<AvatarSettings> GetAsync()
+    {
+        var path = Path.Combine(this.avatarContextService.Avatar.Id, Constants.Filenames.VariablesEnv);
+        var avatarSettings = new AvatarSettings();
+
+        var envFile = await File.ReadAllLinesAsync(path);
+
+        foreach (var line in envFile)
+        {
+            if (string.IsNullOrWhiteSpace(line) || line.Trim().StartsWith("#"))
+                continue;
+
+            var index = line.IndexOf('=');
+
+            // If no "=" found, go to the next line
+            if (index == -1) continue;
+
+            var key = line[..index].Trim();
+            var value = line[(index + 1)..].Trim();
+
+            switch (key.ToUpper())
```

Comment:
I'm looking for a way to shorten these switch statements.

Reply:
Please create a ticket for this in Taiga with references to the classes and files involved.